src/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -41,6 +41,7 @@ const std::vector<std::vector<size_t >> kernelsH1 = {{1, 3},
                                                      {1, 5}};
 const std::vector<std::vector<size_t >> stridesH1 = {{1, 1},
                                                      {1, 3}};
+const std::vector<std::vector<size_t>> stridesH1Big = {{1, 16}};
```

Comment:
Maybe testing stride != multiple of 8 would be beneficial?

Reply:
Added